tomcatWarDeployer.py

```diff
@@ -491,12 +491,17 @@ def preparePayload(opts):
 <%%@page import="java.net.*"%%>
 <%%!
 	public String execute(String pass, String cmd) {
-		final String hardcodedPass = "%(password)s";
+		final String hardcodedPass = "password";
```

Comment:
What is the meaning of this change? :)

Reply:
Sorry, it was due tests on my local tomcat deplyement.
I left it there :')